src/js/custom-modal.js

```diff
@@ -0,0 +1,24 @@
+// Custom Modal toggle script for when we need to use something homegrown
+// instead of the built in bootstrap modal
+const customModalOnClick = (e) => {
+  e.preventDefault();
+
+  const $toggle = $(e.target);
+  const $target = $($toggle.data('target'));
+
+  if (!$target) {
+    return;
+  }
+
+  if ($target.hasClass('show')) {
```

Comment:
Can you use the toggleClass method here instead: http://api.jquery.com/toggleClass/